core/guests.core.js

```diff
@@ -86,7 +86,7 @@ async function findGuestById(guestId) {
  */
 async function saveNewGuestWithEventId(userId, eventId, guestInfo) {
   const {
-    email, phoneNumber, gender, major, something,
+    email, fullName, phoneNumber, gender, major, answer,
```

Comment:
Remove major please

Reply:
`answers` not `answer`

Reply:
solved https://github.com/EasyEventV2/ee2_api/pull/12/commits/05d70661939a65148fa8a6024756c7fe05d6ba49

Reply:
> Remove major please

https://github.com/EasyEventV2/ee2_api/pull/12/commits/87a84ff45ece9b7540cb6d699f210ed1aa795eb8